ergo/conditions/crossentropy.py

```diff
@@ -25,3 +29,46 @@ def structure(cls, params):
 
     def __str__(self):
         return "Minimize the cross-entropy of the two distributions"
+
+
+class PartialCrossEntropyCondition(condition.Condition):
```

Comment:
I can't seem to find where this is actually used -- where is that?

Reply:
Not used anymore - this was from my first attempt at open bounds.